src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -64,13 +64,47 @@ class BeamSkimmer @Inject()(
     }
   }
 
+  def setBeamServices(newBeamServices: BeamServices) = {
+    beamServicesOpt = Some(newBeamServices)
+  }
+
+  def getSkimDefaultValue(
+    mode: BeamMode,
+    origin: Location,
+    destination: Location,
+    departureTime: Int,
+    vehicleTypeId: Id[BeamVehicleType],
+    beamServices: BeamServices
+  ): Skim = {
+    val (travelDistance, travelTime) = distanceAndTime(mode, origin, destination)
+    val travelCost: Double = mode match {
+      case CAR | CAV =>
+        DrivingCost.estimateDrivingCost(
+          new BeamLeg(
+            departureTime,
+            mode,
+            travelTime,
+            new BeamPath(null, null, None, null, null, travelDistance)
+          ),
+          vehicleTypeId,
+          beamServices
+        )
+      case RIDE_HAIL =>
+        beamServices.beamConfig.beam.agentsim.agents.rideHail.defaultCostPerMile * travelDistance / 1609.0 + beamServices.beamConfig.beam.agentsim.agents.rideHail.defaultCostPerMinute * travelTime / 60.0
```

Comment:
Could you put 1609 in a variable so that it make sense what this value is supposed to represent?